docs/source/gateway.md

```diff
@@ -45,87 +39,53 @@ Hyperledger Fabric currently supports client application development in three la
-endpoint address and MSPID of the failing node will be included in the `Details` field of the error returned from the gateway
-to the client.  If the `Details` field is empty, then the error would have originated in the gateway.
+states and will override the chaincode policy or collection policy for private data collection states. The endorsement policies themselves are stored in the ledger and can be updated by new transactions.
+
+The combination of endorsement policies to be applied to the transaction proposal is determined at chaincode runtime and cannot necessarily be derived from static analysis.
+
+The Fabric Gateway manages the complexity of transaction endorsement on behalf of the client, using the following process:
+
+- The Fabric Gateway selects the endorsing peer from the gateway peer's organization (MSPID) by identifying the (available) peer with the highest ledger block height. The assumption is that all peers within the gateway peer's organization are *trusted* by the client application that connects to the gateway peer.
+- The transaction proposal is simulated on the selected endorsement peer. This simulation captures information about the accessed states, and therefore the endorsement policies to be combined (including any individual state-based policies
+stored on the endorsement peer's ledger).  
+- The captured policy information is assembled into a `ChaincodeInterest` protobuf structure and passed to the discovery service in order to derive an endorsement plan specific to the proposed transaction.
+- The gateway applies the endorsement plan by requesting endorsement from the organizations required to satisfy all policies in the plan. For each organization, the gateway peer requests endorsement from the (available) peer with the highest block height.
+
+The gateway is dependent on the [discovery service](discovery-overview.html) to get the connection details of both the available peers and ordering service nodes, and for calculating the combination of peers that are required to endorse the transaction proposal. The discovery service must therefore always remain enabled on the Fabric Gateway peer.
+
+The gateway endorsement process is more restrictive for transient data because it often contains sensitive or personal information that must not persist in the ledger. The gateway sends proposals involving transient data to only the organizations which write the specific transient data to their private collections. The gateway restricts the endorsing organizations to only those that write the specific transient data to a shared private data collection. If the restriction for transient data results in failed endorsement, the gateway rejects the transaction proposal. In this situation, the client should [explicitly define which organizations should endorse](#how-to-override-endorsement-policy-detection) the transaction, which must be limited to organizations that share the relevant private data collection.
```

Comment:
> The gateway sends proposals involving transient data to only the organizations which write the specific transient data to their private collections.

This sentence is written twice here.  It's not entirely correct.  The gateway will restrict the set of endorsing organizations to those that are members of the collections that are accessed (either read or write).  This is different because a collection can be configured to allow 'blind writes' (i.e. written by orgs that aren't members of the collection, and typically the access control is coded in the chaincode.  If this is the case, then client will need to explicitly set the endorsing orgs.

> which must be limited to organizations that share the relevant private data collection

Again, this is not true in the 'blind write' scenario

Reply:
Okay so it seems the concept of being a "member of a collection" is an established one and should remain. ? ... In other words, to a new reader it's a bit odd right, for an org to "belong to / be a member of" to a data collection. ... As opposed to an org. sharing, or subscribing to, or being a co-owner of, a data collection. LMK. @andrew-coleman @denyeart

Reply:
@joshhus I'm not following the question. An organization can indeed be a member of a collection. That means the organization stores the private data and their peers can access it.

Reply:
It's good if you and Andy are good. I just don't follow the logic of org membership/belonging to a data collection yet. I would have thought an org "shares" a common data collection, or "co-owns" a private data collection, but "membership" in data is new concept. BTS e.g. describes org membership in a network, or a channel, e.g.